src/guide/code-owners.md

```diff
@@ -152,7 +140,6 @@ This list is intended to make it easier to identify which email list to include
 * `jdk.httpserver` – Net
 * `jdk.incubator.foreign` – LangTools
 * `jdk.incubator.httpclient` – Net
-* `jdk.incubator.jpackage` – Client
```

Comment:
I assume this directory remains in some of the update release trains still being maintained? If so it should still be in the list.

Reply:
No. It was incubator only in JDK 14 and 15. I see no reason to continue to list it here.

Reply:
It looks to me like both JDK 14 and JDK 15 is being actively maintained by the community.